### ISSUE TITLE
clang parse skips regions of code

### ISSUE THREAD

**Issue by rajukv** [Andersbakken/rtags#759]
This is not a rtags specific issue, but more of a libclang issue. I have noticed that clang skips entire sections of code when parsing. Because of this, rtags doesnt seem to have any tags in the area that is skipped. I have noticed this happening between 2 versions of clang. Things which used to work fine with 3.6.0 stops working in 3.8.0 etc. Has anyone seen issues like this?

Thanks,
Raju

**Comment by Andersbakken** [comment]
Sorry about the delay. I can't say that I've seen it. Are you able to reliably reproduce it? Could you send a small example that demonstrates something working in 3.6 and not in 3.8?

**Comment by rajukv** [comment]
I will get some sample code to reproduce this issue. Currently I see this
issue with some code at work.

**Comment by Andersbakken** [comment]
That would be nice. I'd like to take a look at it.

Anders

**Comment by rajukv** [comment]
For some reason, rdm does not report all the missing header files when processing compile_commands.json. When I ran each of the commands individually, it reports more missing header files. Once I fixed those issues, I dont see this issue anymore. Not sure if something changed between clang 3.6(where this was working) and 3.8. Anyways, I will fix the include directories manually and continue. I will close this issue for now.
